changelog/add-overview-active-disputes-notice

```diff
@@ -0,0 +1,4 @@
+Significance: minor
+Type: add
+
+Display a notice on the Payments → Overview screen when there are active disputes that require a response.
```

Comment:
Adding this changelog entry with the intention that it will likely cover all issues in #6353